lapack.go

```diff
@@ -127,3 +127,12 @@ const (
 	Scale        Job = 'S'
 	PermuteScale Job = 'B'
 )
+
+// Jobs and Comps for Dhseqr.
+const (
+	EigenvaluesOnly     Job = 'E'
```

Comment:
Better(=shorter) names here?

It seems that there arises some conflicting, or at least confusing, naming here. There is
`EigValueOnly EigComp = 'N'`
above from symmetric eigenvalue problems, here we have
`EigenvaluesOnly Job = 'E'` ...